packages/parser/lib/css_lexer.re

```diff
@@ -28,9 +28,7 @@ let unicode_range = [%sedlex.regexp?
   (up_to_6_hex_digits, '-', up_to_6_hex_digits)
 ];
 
-let escape = [%sedlex.regexp?
-  unicode | ('\\', Compl('\r' | '\n' | '\012' | hex_digit))
-];
+let escape = [%sedlex.regexp? '\\'];
```

Comment:
Why this regex has changed?

Reply:
The other parts were redundant and caused issues while starting with escape chars

Reply:
👍🏼